### PR TITLE
Add historical sig-node agenda archive

### PR DESCRIPTION
**Pull request by tallclair**
Auto generated from https://docs.google.com/document/d/1Ne57gvidMEWXR70OxxnRkYquAoMpt56o75oZtg-OeBg/edit# using an apps script. The formatting isn't perfect, but I thought it was close enough for archival purposes.

Note that I omitted today's (Jan 08) meeting. I recommend we delete everything prior to Jan 08 in the existing agenda doc, and push yearly updates here.

/sig node
/assign @dchen1107 @derekwaynecarr

### CODE REVIEW

File: sig-node/archive/weekly-agenda-2017-2018.md [hunk 1]
```diff
@@ -0,0 +1,1385 @@
...
+*   [Sandboxes API Proposal](https://docs.google.com/document/d/1WzO_QjJFfedhsiBtfcVB2QzTWRXHEPX1xOyqDGXxO-0/edit#) (@tallclair)
+    *   Q: Do we expect cluster admin to label nodes which support sandbox?
+        *   It's not clear whether we should require all container runtimes to support sandbox, or this is an optional feature.
+        *   This doesn't block alpha.
+        *   Ideally each node will only support one sandbox technology, but if the user want to use different sandbox technology, they may need to have different node setup and label the node.
+    *   Q: Will this be included in conformance test?
+    *   Q: Can the sandbox be a runc implementation which meets the conformance?
+        *   It is hard to define a conformance test suite to validate whether the sandbox meets the requirement.
+        *   We can only provide guideline for now.
+        *   We are still working on the definition, but one probably defination: idependent kernel, and 2 layers of security?
```
Comment (cblecker) [inline sig-node/archive/weekly-agenda-2017-2018.md]:
```
W0108 20:52:35.970] sig-node/archive/weekly-agenda-2017-2018.md:410:69: "defination" is a misspelling of "definition"
```

File: sig-node/archive/weekly-agenda-2017-2018.md [hunk 2]
```diff
@@ -0,0 +1,1385 @@
...
+    *   Switch to zoom next year
+*   Next Year:
+    *   Secure Container/Pod
+    *   Node level debuggability
+    *   Windows Container.
+    *   Container Runtime Interface
+        *   Container Monitoring.
+        *   Container Logging.
+        *   Secure Container/Pod.
+    *   More? Come back on Janurary.
```
Comment (cblecker) [inline sig-node/archive/weekly-agenda-2017-2018.md]:
```
W0108 20:52:35.970] sig-node/archive/weekly-agenda-2017-2018.md:769:27: "Janurary" is a misspelling of "January"
```

File: sig-node/archive/weekly-agenda-2017-2018.md [hunk 3]
```diff
@@ -0,0 +1,1385 @@
...
+        *   frakti: not relavant anyway
+        *   Windows container support: not relavant
+*   Containerd integration status update https://github.com/kubernetes-incubator/cri-containerd
+    *   Cut v1.0.0-alpha.0 release this week.
+    *   Missing container metrics (pending PR under review, merging this week).
+    *   Image gc is not supported in containerd yet, will be added later.
+        *   Containerd does provide functions to remove underlying storage of image, but the interface is relatively low level, which is not easy enough to use.
+        *   Containerd is adding garbage collection support, which will be much easier to consume.
+        *   CRI-Containerd image gc will build on top of containerd garbage collection.
+    *   User guide on how to create cluster with containerd is avaliable: https://github.com/kubernetes-incubator/cri-containerd/blob/master/contrib/ansible/getting-started.md
```
Comment (cblecker) [inline sig-node/archive/weekly-agenda-2017-2018.md]:
```
W0108 20:52:35.970] sig-node/archive/weekly-agenda-2017-2018.md:985:63: "avaliable" is a misspelling of "available"
```

**Comment by tallclair** [comment]
I switched to the yearly notes format [used in sig-auth](https://github.com/kubernetes/community/pull/3074), and dredged up some more of the archives prior to mid-2017. Also fixed a bunch of spelling errors, and cleaned up whitespace.

**Comment by derekwaynecarr** [comment]
impressive!

**Pull request merged**